pkg/asset/cluster/tfvars.go

```diff
@@ -92,6 +93,7 @@ func (t *TerraformVariables) Dependencies() []asset.Asset {
 		&machines.Master{},
 		&machines.Worker{},
 		&baremetalbootstrap.IronicCreds{},
+		&tls.RootCA{},
```

Comment:
Why is this being added as a dependency?

Reply:
Good point @kirankt can you remember if/why this is required?

Reply:
I suspect it's not required since the masterIgn assed already has this dep ref https://github.com/openshift/installer/blob/master/pkg/asset/ignition/machine/master.go#L32

I'll re-test without it and remove provided everything still works :)

Reply:
We don't need this. It is a mistake.

Reply:
Now removed.